llvm/docs/Telemetry.rst

```diff
@@ -0,0 +1,246 @@
+===========================
+Telemetry framework in LLVM
+===========================
+
+.. contents::
+   :local:
+
+.. toctree::
+   :hidden:
+
+===========================
+Telemetry framework in LLVM
```

Comment:
This header appears to be a duplicate of the one at the top of the file?

Reply:
done